app/src/main/java/org/mozilla/fenix/nimbus/NimbusExperimentsFragment.kt

```diff
@@ -5,81 +5,119 @@
 package org.mozilla.fenix.nimbus
 
 import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
-import android.widget.TextView
-import androidx.core.view.isVisible
+import android.view.ViewGroup
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.findNavController
-import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.recyclerview.widget.RecyclerView
-import kotlinx.coroutines.Dispatchers.IO
-import kotlinx.coroutines.Dispatchers.Main
-import kotlinx.coroutines.launch
-import mozilla.components.service.nimbus.ui.NimbusExperimentAdapter
-import mozilla.components.support.base.log.logger.Logger
+import org.mozilla.experiments.nimbus.AvailableExperiment
 import org.mozilla.fenix.R
 import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.runIfFragmentIsAttached
 import org.mozilla.fenix.ext.showToolbar
-import org.mozilla.fenix.nimbus.view.NimbusExperimentsView
+import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * Fragment use for managing Nimbus experiments.
  */
 @Suppress("TooGenericExceptionCaught")
-class NimbusExperimentsFragment : Fragment(R.layout.mozac_service_nimbus_experiments) {
-
-    private var adapter: NimbusExperimentAdapter? = null
-
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-        bindRecyclerView(view)
-    }
+class NimbusExperimentsFragment : Fragment() {
```

Comment:
Let's do 2 separate PRs - 1 for the Nimbus Branches and 1 for the Nimbus Experiments. I would prefer if we focused on converting Experiments first. Please don't do both at the same time for easier reviews and let's take a more iterative approach here. I do not believe in future iteration we would need to have 2 different fragments because we should be able just add a `NavHost` and navigate between the 2 different screens within the same fragment. See https://developer.android.com/jetpack/compose/navigation